### PR TITLE
added questions and result page

### PR DESCRIPTION
**Pull request by sanjay-xdr**
added: rules page
added: questions page (5 questions)
added:result page with wrong and correct answer
demo:-https://naughty-swirles-d2fa11.netlify.app/pages/rules

### CODE REVIEW

File: Pages/Questions/Q1/question.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  cursor: pointer;
+  height: 8rem;
+  width: 92%;
+  font-size: 2.5rem;
+  color: var(--text-color);
+  padding: 6px 8px;
+
+  border: 1px solid var(--card-background);
+  border-radius: 3rem;
+  background-color: #1f2937;
```
Comment (tushargoyal812) [inline Pages/Questions/Q1/question.css:33]: use color variable here

**Review by tushargoyal812 — COMMENTED** [review]
just minor changes and good to merge.

File: Pages/Questions/Q1/question.css (hunk 1 above)
Comment (sanjay-xdr) [reply Pages/Questions/Q1/question.css:33]: sure

File: Pages/rules.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+.nav-link {
+  text-decoration: none;
+  color: var(--text-color);
+  font-size: 2rem;
+  padding: 1rem;
+}
+
+.border-solid {
+  border: 1px solid #64748b;
```
Comment (devananth) [inline Pages/rules.css]: you can use `css variables` here too

File: Pages/rules.css [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  font-size: 3rem;
+  margin: var(--margin-sm);
+  color: var(--grey-color);
+}
+
+.quiz-start-btn {
+  height: 5rem;
+  width: 35rem;
+  border: none;
+  outline: none;
```
Comment (devananth) [inline Pages/rules.css]: Don't give `outline:none` which affects **accessibility**. Instead you can use 

           outline: 2px solid transparent;
           outline-offset: 2px;

File: UtilityCSS/utility.css [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  border: none;
+}
+
+.border-radius-50 {
+  border-radius: 50%;
+}
+
+.border-solid {
+  border: 2px solid #000000;
+}
```
Comment (devananth) [inline UtilityCSS/utility.css]: you have alerady declared this class previously in `rules.css` . If it is same you can keep it in **common css file**

File: style.css [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+  color: var(--text-color);
+  font-size: 2rem;
+  padding: 1rem;
+}
+.nav-play:hover {
+  color: var(--grey-color);
+}
+.border-solid {
+  border: 1px solid #64748b;
+}
```
Comment (devananth) [inline style.css]: Here too , repeated `css class`

**Review by devananth — CHANGES_REQUESTED** [review]
Good work , Sanjay ! 

Suggested few changes , have a look at them.

File: Pages/rules.css (hunk 3 above)
Comment (sanjay-xdr) [reply Pages/rules.css]: didnt knew that will implement this with other suggested feedbacks

File: Pages/Questions/Q1/question.css [hunk 6]
```diff
@@ -0,0 +1,62 @@
+@import url(../../rules.css);
+@import url(../../../UtilityCSS/utility.css);
+
+.question-container {
+  margin: 5rem;
+}
+
+.question-box {
+  height: auto;
```
Comment (SRUJANA-PENUGONDA13) [inline Pages/Questions/Q1/question.css]: It is not required once verify it does not make any change even removing it

**Comment by SRUJANA-PENUGONDA13** [comment]
Try to maintain equal spacing between header and main container for all pages so that it looks good. Remaining all looks good.  Keep up good work

Ex - for questions page it is good but for rules page it is very less

**Pull request merged**